x/gov/types/query.pb.go

```diff
@@ -1,5 +1,5 @@
 // Code generated by protoc-gen-gogo. DO NOT EDIT.
-// source: cosmos/gov/v1beta1/query.proto
+// source: cosmos/gov/v1beta2/query.proto
```

Comment:
Master has a `types/v1beta2` folder with these same .pb.go files

The desired output folder is `types`, right? In this case we should remove that folder, and also change the output option in proto files.

Reply:
Yup, I thought I had already moved it to the `types` directory, but seems like there is still the `v1beta2` directory